daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

```diff
@@ -348,6 +364,100 @@ curl -X POST http://localhost:3500/v1.0/publish/myKafka/myTopic?metadata.correla
       }'
 ```
 
+## Avro Schema Registry serialization/deserialization
+You can configure publishers/subscribers to publish/consume data encoded using [Avro binary serialization](https://avro.apache.org/docs/), leveraging an [Apache Schema Registry](https://developer.confluent.io/courses/apache-kafka/schema-registry/) (e.g. [Confluent Schema Registry](https://developer.confluent.io/courses/apache-kafka/schema-registry/), [Apicurio](https://www.apicur.io/registry/))
+
+### Configuration
+- The kafka pubsub component metadata must at least have the schema registry URL defined, and API key/secret if applicable.
+- Only message value serialization/deserialization is supported at the moment.
+- Cloud events are not supported. Therefore, the `rawPayload=true` metadata must be passed.
+- Schema subjects are automatically derived from topic names, using standard naming convention:
+  - e.g. given topic `my-topic`, the schema subject will be `my-topic-value`
+- When interacting with the message payload within the service, it will be in JSON format. The payload will be transparently serialized/deserialized within the Dapr component.
```

Comment:
```suggestion
When interacting with the message payload within the service, it will be in JSON format. The payload will be transparently serialized/deserialized within the Dapr component.
```